src/tests/pools/D3MAavePool.t.sol

```diff
@@ -249,28 +247,30 @@ contract D3MAavePoolTest is D3MPoolBaseTest {
         assertEq(D3MAavePool(d3mTestPool).redeemable(), address(adai));
     }
 
-    function test_transfer_adai() public {
+    function test_exit_adai() public {
```

Comment:
since we now don't expect this to be called except during ES, maybe we can remove this test? I think it is a hold over from when `transfer` was for upgrading too

Reply:
To be honest I think the one that doesn't make sense is the vat caged one. This is a unit test for the `pool` and the `vat` status has nothing to do with this method.

Reply:
actually, you're right.  Lets remove the cage one?